ldi-orchestrator/ldio-connectors/ldio-http-in-poller/src/test/java/be/vlaanderen/informatievlaanderen/ldes/ldio/HttpInputPollerTest.java

```diff
@@ -0,0 +1,149 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.config.ComponentProperties;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.services.ComponentExecutor;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.config.HttpInputPollerAutoConfig;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.exceptions.InvalidPollerConfigException;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.function.Executable;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+import org.mockito.Mockito;
+
+import java.util.Map;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.stream.Stream;
+
+import com.github.tomakehurst.wiremock.client.CountMatchingStrategy;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
+
+import static be.vlaanderen.informatievlaanderen.ldes.ldio.config.HttpInputPollerProperties.*;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@WireMockTest(httpPort = 10101)
+class HttpInputPollerTest {
+	private LdiAdapter adapter;
+	private static final String BASE_URL = "http://localhost:10101";
+	private static final String ENDPOINT = "/resource";
+	private ComponentExecutor executor;
+	private HttpInputPoller httpInputPoller;
+	private ScheduledExecutorService scheduledExecutorService;
+
+	private static final String CONTENT = "_:b0 <http://schema.org/name> \"Jane Doe\" .";
+	private static final String CONTENT_TYPE = "application/n-quads";
+
+	private static ComponentProperties createConfig(String url, String interval, String continueOnFail) {
+		return new ComponentProperties(Map.of(URL, url, INTERVAL, interval, CONTINUE_ON_FAIL, continueOnFail));
+	}
+
+	private static ComponentProperties createDefaultTestConfig() {
+		return createConfig(BASE_URL + ENDPOINT, "PT1S", "false");
+	}
```

Comment:
I'd recommend to order this a bit more. 
First all the static finals, then the non finals,..

And then below all your tests, the private methods/functions